### PR TITLE
[SASS-2939] Pensions - annual and lifetime allowance gateway page

### PR DESCRIPTION
**Pull request by adam-shahzad-2**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-CHANGE-ME](ADD URL HERE)

##### Before Reviewing
- [X]  Have you pulled the branch down?
- [X]  Have you assigned yourself to the PR?
- [X]  Have you moved the task to “in review” on JIRA?
- [X]  Have you checked to ensure all dependencies are up to date?
- [X]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [ ]  Have you run the journey tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of main?
- [x]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [ ]  Have you checked the PR Builder passes?

### CODE REVIEW

**Comment by adam-shahzad-2** [comment]
so the welsh error messages are wrong, 
image
image
Should I just change them to english, and merge in?

**Comment by jhkldev** [comment]
@adam-shahzad-2 You need to confirm with design what the text should be.

File: app/controllers/pensions/lifetimeAllowance/AnnualLifetimeAllowanceGatewayController.scala [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import scala.concurrent.{ExecutionContext, Future}
+
+class AnnualLifetimeAllowanceGatewayController @Inject()(authAction: AuthorisedAction,
+                                                         view: AnnualLifetimeAllowanceGatewayView,
+                                                         pensionSessionService: PensionSessionService,
+                                                         errorHandler: ErrorHandler)
+                                                        (implicit cc: MessagesControllerComponents, appConfig: AppConfig, ec: ExecutionContext)
+  extends FrontendController(cc) with I18nSupport with SessionHelper {
+
+  def unauthorisedPaymentForm(isAgent: Boolean): Form[Boolean] = YesNoForm.yesNoForm(
```
Comment (jhkldev) [inline app/controllers/pensions/lifetimeAllowance/AnnualLifetimeAllowanceGatewayController.scala:44]: The name needs updating here. It should be something like `annualLifetimeAllowanceForm`

**Review by jhkldev — CHANGES_REQUESTED** [review]

File: app/controllers/pensions/lifetimeAllowance/AnnualLifetimeAllowanceGatewayController.scala (hunk 1 above)
Comment (adam-shahzad-2) [reply app/controllers/pensions/lifetimeAllowance/AnnualLifetimeAllowanceGatewayController.scala:44]: eek, wonder where that came from

**Review by jhkldev — APPROVED** [review]

**Pull request merged**